google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/application/AppEngineAdminService.java

```diff
@@ -29,9 +29,8 @@
 import java.util.List;
 
 /**
- * A Service that handles interactions with the App Engine Admin API. See
- * <a href="https://cloud.google.com/appengine/docs/admin-api/">
- *   https://cloud.google.com/appengine/docs/admin-api/</a>
+ * A service that handles App Engine Administration. This class provides some general caching logic,
+ * but delegates to extending classes to performing the actual administrative actions.
  */
 public abstract class AppEngineAdminService {
```

Comment:
I think this is another merge problem. This was updated in a different PR. Could you update these to make the PR easier to review?